awx/api/views/organization.py

```diff
@@ -181,6 +181,14 @@ class OrganizationNotificationTemplatesAnyList(SubListCreateAttachDetachAPIView)
     relationship = 'notification_templates_any'
 
 
+class OrganizationNotificationTemplatesStartedList(SubListCreateAttachDetachAPIView):
+
+    model = NotificationTemplate
+    serializer_class = NotificationTemplateSerializer
+    parent_model = Organization
+    relationship = 'notification_templates_started'
+
+
```

Comment:
If we're gonna move to new pattern, let's do the same thing with `OrganizationNotificationTemplatesAnyList`, `OrganizationNotificationTemplatesStartedList`, `OrganizationNotificationTemplatesErrorList `

Reply:
Oops, missed these.  They've been changed.  I searched the codebase for other instances and it seems like this was the last bit.